### DISCUSSION TITLE
How do I add a mod to this repository?

### DISCUSSION THREAD

**Discussion by Anixx** [ramensoftware/windhawk-mods#491] [Q&A]
Should I fork this repository altogether first?

**Comment by m417z** [comment]
Yes. The instructions can be found here:
https://github.com/ramensoftware/windhawk-mods?tab=readme-ov-file#submitting-a-new-mod

If creating pull requests is new to you, check out GitHub docs:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request
